tests/scripts/thread-cert/border_router/test_publish_meshcop_service.py

```diff
@@ -95,7 +95,26 @@ def test(self):
         br1.start()
         self.simulator.go(config.BORDER_ROUTER_STARTUP_DELAY)
         self.assertEqual('leader', br1.get_state())
+
+        # get initial ephemeral key mode (ePSKc) status
+        epskc_enabled = br1.ephemeral_key_enabled
+        # TODO: default status is True when OT version >= 1.4
+        self.check_meshcop_service(br1, host)
+
+        # change ephemeral key mode (ePSKc) and check Meshcop
+        epskc_enabled = not epskc_enabled
```

Comment:
Instead of doing this, I'd suggest to test it more deterministically. 

That is, test the disabled scenario then test the enabled scenario.

Reply:
It's flexible for checking both enabled/disabled as we are not sure the initial state, how about adding log for the real state?

Reply:
Has changed test to set True/False directly now. and change back to original state incase there are other test scenarios later